test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/CompositeModelBinderTest.cs

```diff
@@ -65,6 +65,7 @@ public class CompositeModelBinderTest
             var bindingContext = new ModelBindingContext
             {
                 FallbackToEmptyPrefix = true,
+                IsTopLevelObject = true,
```

Comment:
Right but that's not new. We should _never_ have fallen back unless `IsTopLevelObject == true`.